_posts/2022-11-25-checking-in-on-dawson.md

```diff
@@ -0,0 +1,57 @@
+---
+date: 2022-11-25
```

Comment:
leaving this comment as a reminder to update the pub date once we have a pub date selected; update the filename to match